back/app/models/idea.rb

```diff
@@ -159,6 +159,10 @@ class Idea < ApplicationRecord
       .where('ideas.id NOT IN (SELECT DISTINCT(post_id) FROM official_feedbacks)')
   }
 
+  scope :draft_surveys, lambda {
```

Comment:
Does it include phygital ideas? If so, I guess we don't need to create activities for them.

Reply:
Yes imported phygital ideas are also draft - whether they are surveys or ideas. Guess a side effect here will be that draft surveys will appear in that list to be signed off. Hmm...